UT4MasterServer/Controllers/AccountController.cs

```diff
@@ -97,6 +97,21 @@ public async Task<IActionResult> GetAccount(string id)
 		obj.Add("cabinedMode", false);
 		obj.Add("hasHashedEmail", false);
 
+		var flags = new List<string>();
+
+		var flagNamesAll = Enum.GetNames<AccountFlags>();
+		var flagValuesAll = Enum.GetValues<AccountFlags>();
+
+		for (int i = 0; i < flagNamesAll.Length; i++)
+		{
+			if (account.Flags.HasFlag(flagValuesAll[i]))
+			{
+				flags.Add(flagNamesAll[i]);
+			}
+		}
+		var flagsArray = new JArray(flags);
+		obj.Add("flags", flagsArray);
+

```

Comment:
This endpoint is used by the game and should not be changed. From the looks of it, it seems that what you added here is never even needed anywhere.

Reply:
I use it in the front end, but I can put it in another place. Persona controller?